sinonjs/sinon
# Rename History.md to CHANGELOG.md

#### Purpose (TL;DR)
Fix issue #1947 by renaming History.md to CHANGELOG.md


 #### Background (Problem in detail)
CHANGELOG.md is pretty much the de facto standard for where to put the changelog. While it might seem obvious that History.md would be a place to look, when going over a very large amount of dependencies in our project, I wasn't looking for alternative names, just variations of "changelog", as well as looking at the Releases tab on GitHub and looking through the readme for a changelog-section. It actually took me several minutes to find the right file.


#### Solution
I did a project-wide search for `History.md`, and it seemed to me like all instances should be replaced, so that's what I did, apart from renaming the file of course.


 #### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. `npm run preversion`
4. `npm run postversion`

 #### Checklist for author
